Model/src/CardPackage/Card.java

```diff
@@ -0,0 +1,4 @@
+package CardPackage;
+
+public class Card {
+}
```

Comment:
Do we want an overall Card class or ChanceCard. Isnt there multple types of cards?

Reply:
There are two types of cards (Chance and Community Chest) but at the end of the day, both card types do the exact same thing: Get drawn from a pile and then trigger an action in game. When I made this card class, I made it on the assumption that we could just get away with having an instance variable inside the class that determines what type of Card it is.